src/Appwrite/Utopia/Response/Model/Session.php

```diff
@@ -53,10 +53,10 @@ public function __construct()
                 'example' => 'MTQ0NjJkZmQ5OTM2NDE1ZTZjNGZmZjI3',
             ])
             ->addRule('providerAccessTokenExpiry', [
-                'type' => self::TYPE_INTEGER,
-                'description' => 'Date, the Unix timestamp of when the access token expires.',
-                'default' => 0,
-                'example' => 1592981250,
+                'type' => self::TYPE_DATETIME,
+                'description' => 'The date of when the access token expires in datetime format.',
```

Comment:
I would use either DateTime or Datetime consistently, but not "datetime".

Reply:
Done!